packages/@react-spectrum/progress/stories/ProgressBar.stories.tsx

```diff
@@ -130,6 +130,38 @@ storiesOf('Progress/ProgressBar', module)
       );
     }
   )
+  .add(
+    'parent width 100%',
+    () => (
+      <span style={{width: '100%'}}>
+        {render()}
+      </span>
+    )
+  )
+  .add(
+    'parent width 100px',
+    () => (
+      <span style={{width: '100px'}}>
+        {render()}
+      </span>
+    )
+  )
+  .add(
+    'width: 300px',
+    () => render({width: '300px'})
+  )
+  .add(
+    'width: 300px, labelPosition: side, isIndeterminate: true',
+    () => render({width: '300px', labelPosition: 'side', isIndeterminate: true})
+  )
+  .add(
+    'width: 30px',
+    () => render({width: '30px'})
+  )
+  .add(
+    'width: 30px, labelPosition: side, isIndeterminate: true',
+    () => render({width: '30px', labelPosition: 'side', isIndeterminate: true})
```

Comment:
I think the css for the `labelPosition: side` will need to account for the case in which the width applied is less than the `min-width`. At the moment the wrapping div (`.spectrum-BarLoader`) will be of width 192px, but that doesn't account for the label width and thus the following can happen when there are sibling elements (aka the button is hidden by the progress bar):
![image](https://user-images.githubusercontent.com/9661127/107448663-f6337a80-6af6-11eb-973e-915460201696.png)

Reply:
I think spectrum-css styles might suffer from the same issue. Maybe a `min-width: fit-content` and have the label have a max width so it calculates properly? Open to opinions for this one